api/v1beta1/flowcollector_types.go

```diff
@@ -735,7 +736,6 @@ type FlowCollectorStatus struct {
 // +kubebuilder:printcolumn:name="Sampling (EBPF)",type="string",JSONPath=`.spec.agent.ebpf.sampling`
 // +kubebuilder:printcolumn:name="Deployment Model",type="string",JSONPath=`.spec.deploymentModel`
 // +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[*].reason"
-// +kubebuilder:storageversion
```

Comment:
so you must rollback and keep v1beta1 as the storage version, for at least one release. It shouldn't be an issue that the storage version differs from the hub version.

Reply:
Ok. I'll try that. But remember having trouble keeping the hub on the new one and the storage version on v1beta1. Anyway I'll try it.